lib/mailer.js

```diff
@@ -284,7 +285,7 @@ const factory = (options) => {
     precompile: compile,
     render: render,
     send: sendEmail,
-    init
```

Comment:
What's the reason for renaming the public `init()` interface to `initFactory()` here?